### PR TITLE
Add contact-me form

### PR DESCRIPTION
**Pull request by Kajal19-del**
# In this project I did the following activities:
- I create the repository with name "portfolio-project"
- I create the HTML, CSS, the README, gitignore and some other files.
- I add contact-me form
- I using mobile first design
- I add linters to the repository
- I check all the linters errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -209,6 +209,23 @@ <h3 class="skill-language">Skills</h3>
...
+
+  <section class="formx">
+    <div class="contact-form">
+      <h2 class="contact-title">Contact me</h2>
+      <p class="contact-text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it </p>
+      <div class="form">
+        <form action="https://formspree.io/f/mjvzwnzn" method="POST">
+          <input class="form-input" type="text" placeholder="Enter your name" maxlength="30" required>
+          <input class="form-input" type="email" placeholder="Enter your Email" required>
+          <textarea class="form-textarea" name="area" id="" cols="30" rows="6" placeholder="Write your message here" required></textarea>
```
Comment (mgmediaweb) [inline index.html]: I kindly suggest you limit your textarea to 500 characters

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @Kajal19-del , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost2

## Highlights 💯 
- Professional-looking readme file. ✔️ 
- Clear and descriptive Pull Request. ✔️ 
- Clean and tidy code. ✔️ 
- Good job following the design. ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Add descriptive title into your portfolio <title>

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -209,6 +209,23 @@ <h3 class="skill-language">Skills</h3>
       </div>
     </section>
   </footer>
+
+  <section class="formx">
+    <div class="contact-form">
+      <h2 class="contact-title">Contact me</h2>
+      <p class="contact-text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it </p>
+      <div class="form">
+        <form action="https://formspree.io/f/mjvzwnzn" method="POST">
```
Comment (SimonGideon) [inline index.html:218]: According to MDN doc provided by microverse in the course work, Is advices to use ul or li in out forms to provide accessibility.
🥳 👎  Good job in implementing the Figma UI which matches you project Kudoz.

File: styles.css [hunk 3]
```diff
@@ -312,3 +312,101 @@ header {
...
+.form-button:hover {
+  background: #6070ff;
+  border: 1px solid #fff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  border-radius: 8px;
+  color: #fff;
+}
+
+.form-button:active {
+  background: #2230d2;
```
Comment (SimonGideon) [inline styles.css:409]: 🎆 Nice and clean CSS code, i like your work. 👍

**Comment by M1rn4** [comment]
Please consider more descriptive class names for future occasions. :+1:  :100:

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Kajal19-del 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - Good UI design. ✔️
 - No linter errors. ✔️  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**